tests/src/lib.rs

```diff
@@ -24,6 +24,7 @@ cfg_if! {
     }
 }
 
+pub mod builders;
```

Comment:
I would like to see an additional test: generate a builder for TestAllTypesProto3 and write that to file. This has two purposes:
- It adds a example of all posible setters to the PR
- It forces changes to builder API to be visible in future PRs.

The repo has examples for a test writing to file and asserting nothing changes: `test_generate_no_empty_outputs`